skflow/estimators/base.py

```diff
@@ -62,12 +62,18 @@ class TensorFlowEstimator(BaseEstimator):
         early_stopping_rounds: Activates early stopping if this is not None.
             Loss needs to decrease at least every every <early_stopping_rounds>
             round(s) to continue training. (default: None)
-
+        max_to_keep: The maximum number of recent checkpoint files to keep.
+            As new files are created, older files are deleted.
+            If None or 0, all checkpoint files are kept.
+            Defaults to 5 (that is, the 5 most recent checkpoint files are kept.)
+        keep_checkpoint_every_n_hours: Number of hours between each checkpoint
+            to be saved. The default value of 10,000 hours effectively disables the feature.
     """
 
     def __init__(self, model_fn, n_classes, tf_master="", batch_size=32, steps=50, optimizer="SGD",
                  learning_rate=0.1, tf_random_seed=42, continue_training=False,
-                 num_cores=4, verbose=1, early_stopping_rounds=None):
+                 num_cores=4, verbose=1, early_stopping_rounds=None,
```

Comment:
You need to add all this options to other estimator's `__init__`... 
I can see why you were asking about creating Saver in a different place. We need to think what we gonna do with this amount of parameters... sklearn's estimators don't have as much...